### PR TITLE
3/media save notif

### PR DESCRIPTION
**Pull request by abea**
Resolves https://apostrophecms.atlassian.net/browse/A30U-465, adding notifications to saving pieces.

### CODE REVIEW

File: modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue [hunk 1]
```diff
@@ -209,29 +213,36 @@ export default {
     submit() {
       this.triggerValidation = true;
       this.$nextTick(async () => {
-        if (!this.docUtilityFields.hasErrors && !this.docOtherFields.hasErrors) {
```
Comment (abea) [inline modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue:212]: View this with the whitespace changes hidden.

**Comment by abea** [comment]
The tests are failing due to an `uploadfs` security vulnerability.

File: modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue [hunk 2]
```diff
@@ -209,29 +213,36 @@ export default {
...
+        } else {
+          route = this.moduleOptions.action;
+          requestMethod = apos.http.post;
         }
+
+        await requestMethod(route, {
+          busy: true,
+          body: this.doc.data
+        });
+        this.$emit('saved');
```
[the remark this answers is not in the document]
Comment (stuartromanek) [reply modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue:244]: yes

**Review by stuartromanek — COMMENTED** [review]
I can't tell if this bug was introduced here, but:

- Open media mgr
- edit some image and save it (get notification!)
- open another image for editing, all subsequent schema views are blank

**Comment by stuartromanek** [comment]
Still getting this bug when flipping among images
image
(all slugs are taken)

**Comment by abea** [comment]
Gnarly, but unrelated.

**Review by stuartromanek — APPROVED** [review]

File: modules/@apostrophecms/schema/ui/apos/components/AposSchema.vue [hunk 3]
```diff
@@ -101,9 +101,12 @@ export default {
         // The doc might be swapped out completely in cases such as the media
         // library editor. Repopulate the fields if that happens.
         if (
-          this.fieldState._id &&
-          (newVal.data._id !== this.fieldState._id.data)
```
Comment (boutell) [inline modules/@apostrophecms/schema/ui/apos/components/AposSchema.vue:105]: This is not a field,  it concerns me that we're treating it as one. But, taht wasn't first introduced here.

**Review by boutell — APPROVED** [review]

**Pull request merged**